schedule/alp/alp_yast_as_workload.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: alp_yast_as_workload
+description: >
+    Enable YaST automation test for ALP
+vars:
+    YUI_REST_API: 1
+schedule:
+    - installation/bootloader_uefi
+    - microos/selfinstall
```

Comment:
for Flavor: Micro-Default seems that we need a different schedule file

Reply:
for Flavor: Micro-Default, I used 'schedule/alp/alp_yast.yaml' schedule file.
Please check the setting: https://openqa.suse.de/tests/11345346#settings